src/keyboard/textinput.js

```diff
@@ -63,21 +80,16 @@ var TextInput = function(parentNode, host) {
         if (options.role) {
             text.setAttribute("role", options.role);
         }     
-    };
-    this.setAriaLabel = function() {
-        if(host.session && host.renderer.enableKeyboardAccessibility) {
-            var row =  host.session.selection.cursor.row;
-
+        if (options.label) {
```

Comment:
yeah that's fair (I originally called it `setAriaLabel` but changed it to just `label` because the other one was too verbose, `setLabel` sounds like a good middleground)